cmake/SetupDependentOptions.cmake

```diff
@@ -9,6 +9,14 @@
 ## Here are the CMake dependent options in RAJA.
 ##
 
+set(RAJA_DEPENDENT_OPTIONS ENABLE_OPENMP ENABLE_CUDA ENABLE_HIP ENABLE_CLANG_CUDA ENABLE_COVERAGE ENABLE_TESTS ENABLE_EXAMPLES ENABLE_BENCHMARKS)
+foreach (option ${RAJA_DEPENDENT_OPTIONS})
+   if (${RAJA_${option}})
+      if (NOT ${option})
+         message(WARNING "RAJA_${option} set to On, but ${option} is Off")
```

Comment:
Maybe beef up the message by adding something about ${option} must be on to enable feature??

Reply:
That's a good idea - so change the `WARNING` to `FATAL_ERROR` unless `RAJA_ALLOW_INCONSISTENT_OPTIONS` is set.

What do we want the default value for that option to be?